ci/test_notebooks.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# Copyright (c) 2020-2023, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+. /opt/conda/etc/profile.d/conda.sh
+
+rapids-logger "Generate notebook testing dependencies"
+rapids-dependency-file-generator \
+  --output conda \
+  --file_key test_notebooks \
+  --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch);py=${RAPIDS_PY_VERSION}" | tee env.yaml
+
+rapids-mamba-retry env create --force -f env.yaml -n test
+
+# Temporarily allow unbound variables for conda activation.
+set +u
+conda activate test
+set -u
+
+rapids-print-env
+
+rapids-logger "Downloading artifacts from previous jobs"
+CPP_CHANNEL=$(rapids-download-conda-from-s3 cpp)
+PYTHON_CHANNEL=$(rapids-download-conda-from-s3 python)
+
+rapids-mamba-retry install \
+  --channel "${CPP_CHANNEL}" \
+  --channel "${PYTHON_CHANNEL}" \
+  cuspatial libcuspatial
+
+NBTEST="$(realpath "$(dirname "$0")/utils/nbtest.sh")"
+pushd notebooks
+
+# Add notebooks that should be skipped here
+# (space-separated list of filenames without paths)
+SKIPNBS=""
+
+# Set SUITEERROR to failure if any run fails
+SUITEERROR=0
+
+set +e
```

Comment:
Can we adapt this error handling to match https://github.com/rapidsai/cuspatial/pull/912?

You can check out the description and changes in that PR for more details.